src/keygen/participant.rs

```diff
@@ -28,6 +28,18 @@ use rand::{CryptoRng, RngCore};
 use serde::{Deserialize, Serialize};
 use tracing::{info, instrument};
 
+// Storage identifiers for the keygen protocol.
+#[derive(Clone, Copy, Debug, Serialize)]
+enum StorageType {
```

Comment:
Should this be renamed `StorageId` or `StorageIdentifier`?

Reply:
I have a slight preference to keep the name as-is; I prefer `Identifier` to be attached to something that's globally unique.